pkg/pgmodel/migrations/sql/1_base_schema.up.sql

```diff
@@ -1471,3 +1471,38 @@ CREATE VIEW SCHEMA_INFO.label AS
     ARRAY(SELECT value FROM SCHEMA_CATALOG.label l WHERE l.key = lk.key ORDER BY value)
       AS values
   FROM SCHEMA_CATALOG.label_key lk;
+
+
+
+CREATE PROCEDURE SCHEMA_CATALOG.decompress_chunks_after(metric_table NAME, min_time TIMESTAMPTZ)
```

Comment:
Maybe add a comment about why we're doing this in a procedure rather than just calling the function? Note the locking behavior?